File: easy/time_to_eat.lua
for line in io.lines(arg[1]) do
  local a = {}
  for i in line:gmatch("%S+") do a[#a + 1] = i end
  table.sort(a, function (a,b) return (a > b) end)
  print(table.concat(a, " "))
end

File: easy/time_to_eat.clj
(require '[clojure.string :as str])
(defn reverse-cmp [a b]
  (compare b a))
(doseq [item (for [st (remove empty? (str/split-lines (slurp (first *command-line-args*))))]
  (sort reverse-cmp (str/split st #" "))
)] (println (str/join " " item)))
